code/__DEFINES/dcs/signals/signals_assets.dm

```diff
@@ -0,0 +1,2 @@
+/// Sent when an asset datum is fully generated
```

Comment:
Needs the same info as the rest of the signal comments, specifically what the `source` is (I don't care about the call source)